### PR TITLE
Error handling for products

### PR DESCRIPTION
**Pull request by mslup**
- plus some minor corrections

### CODE REVIEW

**Comment by mslup** [comment]
Found one issue, I'll get to it soon

**Comment by mslup** [comment]
Issue resolved itself I guess

File: src/utils/handleError.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { AxiosError } from 'axios';
+import { createStandaloneToast } from '@chakra-ui/react';
+
+const { toast } = createStandaloneToast();
+
+export const handleError = (error: unknown, errorTitle: string): void => {
+  let errorMessage = 'An unexpected error occurred. Please try again.';
```
Comment (andrmdmd) [inline src/utils/handleError.ts:7]: please add 'Please try again' to all error messages (as we do not want users to leave our site!!!)

File: src/pages/product_details/ProductDetails.tsx [hunk 2]
```diff
@@ -90,7 +90,7 @@ const ProductDetails = (): JSX.Element => {
         setProductReviews(response.data);
       }
     } catch (error) {
-      console.error('Error fetching product reviews:', error);
+      handleError(error, 'Error fetching product reviews.');
```
Comment (andrmdmd) [inline src/pages/product_details/ProductDetails.tsx:93]: Same comment as below

**Review by andrmdmd — APPROVED** [review]

**Review by andrmdmd — APPROVED** [review]

**Pull request merged**